### PR TITLE
Entering a valid hex code sets the color

### PR DESCRIPTION
**Pull request by lorcan**
Either pasting a hex value or typing a hex-like value into the hex
input sets the preview value and the value.

Looks like this (first I type in two valid hex strings, then I paste in a valid hex string):

entering-hex-sets-value

### CODE REVIEW

**Comment by lorcan** [comment]
Hi there, anything I can do to progress this PR? (I'm working on an app that has users that encounters this issue fairly regularly.)

**Comment by lukemelia** [comment]
Hi @lorcan. Sorry for the silence on this. This looks like a good improvement to me. Can you add test coverage?

**Comment by lorcan** [comment]
Great, will do @lukemelia, thanks :-)

File: tests/unit/components/col-pick-test.js [hunk 1]
```diff
@@ -79,3 +79,20 @@ test("correctly renders initial bound value", function(assert) {
...
+
+  Ember.run(function() {
+    component.$('.colpick_hex_field input').val('ff000').keyup();
+  });
+  assert.equal(component.get('value'), null, 'After receiving five hex characters we do not yet have a valid color to set');
+
+  Ember.run(function() {
+    component.$('.colpick_hex_field input').val('ff0000').keyup();
+  });
+  assert.equal(component.get('value'), 'ff0000', 'After receiving six hex characters the color is set');
```
Comment (lorcan) [inline tests/unit/components/col-pick-test.js]: This last assertion fails on master.

**Comment by lorcan** [comment]
Hey @lukemelia, sorry about the delay on this. I've added the tests plus a commit to get CI working again. We should be ready to go now :-)

Any thoughts on upgrading the ember version of this plugin? I'd be more than happy to put some time into it :-)

File: bower.json [hunk 2]
```diff
@@ -9,7 +9,7 @@
     "ember-qunit": "0.3.1",
     "ember-qunit-notifications": "0.0.7",
     "ember-resolver": "~0.1.15",
-    "jquery": "^1.11.1",
+    "jquery": "1.11.1",
```
Comment (lorcan) [inline bower.json]: See https://github.com/ember-cli/ember-cli/issues/5316

**Comment by lorcan** [comment]
Hi there, any chance we could get this one merged and get a bump on the version? I've added a test to cover the functionality and fixed CI.

**Comment by lukemelia** [comment]
@lorcan I'm happy to do that. What's the issue with the jQuery version? Is there a version dependency we need to document?

**Comment by lorcan** [comment]
It's an issue with the version of ember-cli being used - there's more details [here](https://github.com/ember-cli/ember-cli/issues/5316) and a good description [here](http://benlimmer.com/2016/01/08/ember-jquery-dependancies/):
 
> Just recently, jQuery 1.12.x was released to bower, and Ember-CLI’s default blueprint allowed upgrading to this version. Unfortunately, all current versions of Ember have a hardcoded check that will make you have a bad time.

It was fixed in ember by https://github.com/emberjs/ember.js/pull/12793.

**Comment by lorcan** [comment]
I'd be happy to produce PRs to update both ember-cli and ember for this project in a bit.

**Comment by lukemelia** [comment]
Sounds good @lorcan. I'll go ahead and cut a release in the meantime.

**Comment by lorcan** [comment]
Awesome!!! Thanks a mil @lukemelia :-D

**Comment by lorcan** [comment]
Hi @lukemelia, any chance you could cut a release to get this out?

**Comment by lukemelia** [comment]
@lorcan published at 0.6.1. Thanks!

**Comment by lorcan** [comment]
Super, thanks @lukemelia!

**Pull request merged**